src/store/reducers.ts

```diff
@@ -273,6 +273,7 @@ export const errorsReducer = handleActions(
 export interface SettingsType {
   locale: string;
   darkMode?: boolean;
+  showBirthdayCalendar?: boolean;
```

Comment:
At the moment it won't show for people by default. You could implement a migration to set a default value for it, but I guess an easier solution would be to flip this field.
So `showBirthdayCalendar` -> `hideBirthdayCalendar` and have the default be `false` which is great because `undefined` is also falsey.